docs/getting-started/integration_rewards.md

```diff
@@ -8,6 +8,22 @@ permalink: /getting-started/integration-rewards/
 
 # Integration rewards
 
-We encourage you to apply for integration rewards (upto **$20,000**) once your project is
-successfully integrated with OSS-Fuzz. Please see the details in our blog post
+We encourage you to apply for integration rewards (up to **$20,000**) once your project
+is successfully integrated with OSS-Fuzz. Please see the details in our blog post
 [here](https://opensource.googleblog.com/2017/05/oss-fuzz-five-months-later-and.html).
+
+Rewards are based on the quality of integration with OSS-Fuzz, which is evaluated using
+the following criteria:
+* Upstream integration of the fuzz targets and build support.
+* Performance of the fuzz targets and code coverage achieved with fuzzing.
+* Regression testing in the upstream repository using fuzz targets and OSS-Fuzz corpora.
+  Enabling [CIFuzz](https://google.github.io/oss-fuzz/getting-started/continuous-integration/)
+  is the easiest way to address this.
+* Discretion bonus to recognize outstanding work.
+
+For each of the points above, the OSS-Fuzz rewards panel first sets up a cap of up to $5,000.
+Then, the panel decides the actual reward amount (randing from $0 up to the cap) for each
```

Comment:
s/randing/ranging/

Reply:
lol, I explicitly used spellchecker but that didn't save me! Thanks!